domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/fradrag/FradragStrategy.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.su.se.bakover.domain.beregning.fradrag
+
+import no.nav.su.se.bakover.domain.beregning.Fradrag
+import no.nav.su.se.bakover.domain.beregning.Fradragstype
+
+sealed class FradragStrategy {
+    abstract fun beregnFradrag(forventetInntekt: Int, fradrag: List<Fradrag>): List<Fradrag>
+
+    object Enslig : FradragStrategy() {
+        override fun beregnFradrag(forventetInntekt: Int, fradrag: List<Fradrag>): List<Fradrag> =
```

Comment:
Alle implementasjonene ser like ut? Kunne vært lagt inn som default da?

Reply:
Det stemmer. Har valgt å legge til placeholders for de ulike variantene i et forsøk på å tydeliggjøre at "her skal de aktuelle endringene legges inn".